### PR TITLE
Add All Laporan feature

### PR DESCRIPTION
**Pull request by adhiiimf**
In this PR, i'll push about all Laporan by PJ practicum in assistant page with privilege role. And i've fixed typo error message at praktikan register. And the last, update assets for assistant avatar.

Thankyou,
**Adhi MF**

### CODE REVIEW

File: resources/js/Pages/Laporan.vue [hunk 1]
```diff
@@ -0,0 +1,489 @@
+<template>
```
Comment (IritaSee) [inline resources/js/Pages/Laporan.vue:1]: TL: DR, but i am sure you are the best, so i put my total trust

**Review by IritaSee — APPROVED** [review]
Nice one, we should create a small bug flag next time, so it is distinguishable from the urgent one

**Comment by IritaSee** [comment]
Okay so this is ready but, still we currently dont have a good launcher script, ping me for merging and deployment

**Comment by adhiiimf** [comment]
> Nice one, we should create a small bug flag next time, so it is distinguishable from the urgent one

Thankyou bang for created this label

**Comment by adhiiimf** [comment]
> Okay so this is ready but, still we currently dont have a good launcher script, ping me for merging and deployment

Okay bang, can i merging this PR now? but i think don't use github actions for this PR

**Pull request merged**